index.yaml

```diff
@@ -53,7 +78,7 @@ entries:
     version: 2.1.1
   - apiVersion: v1
     appVersion: 2.0.3
-    created: "2020-06-25T19:23:19.82968475Z"
+    created: "2020-07-15T09:53:28.449894048Z"
```

Comment:
@desaintmartin we really shouldn't update release dates of older versions. Can you take a look why is it being updated?

Reply:
Indeed...